packages/plugin-ext/src/main/browser/webview/webview.ts

```diff
@@ -308,6 +318,16 @@ export class WebviewWidget extends BaseWidget implements StatefulWidget {
         return this.doSend('did-load-localhost', { origin, location: redirect });
     }
 
+    protected dispatchMouseEvent(type: string, data: MouseEvent): void {
+        const domRect = this.node.getBoundingClientRect();
+        this.node.dispatchEvent(new MouseEvent(type, {
+            ...data, ...{
+                clientX: domRect.x + data.clientX,
+                clientY: domRect.y + data.clientY
+            }
```

Comment:
Don't `...{` but rather just define the options directly after `...data,`?

Reply:
Good catch 👍, Fixed.